src/util/util_misc.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <d3d9types.h>
+
+namespace dxvk {
+
+  inline void DecodeD3DCOLOR(D3DCOLOR color, float* rgba) {
+    // Encoded in D3DCOLOR as argb
+    rgba[3] = (float)((color & 0xff000000) >> 24) / 255.0f;
+    rgba[0] = (float)((color & 0x00ff0000) >> 16) / 255.0f;
+    rgba[1] = (float)((color & 0x0000ff00) >> 8)  / 255.0f;
+    rgba[2] = (float)((color & 0x000000ff))       / 255.0f;
+  }
+
+  template <typename T, size_t n>
+  size_t countof(const T(&)[n]) { return n; }
```

Comment:
Can toss this countof and use std::size instead.

Reply:
this countof was never used, I dropped it.